internal/impl/zeromq/native_input_zmq4n.go

```diff
@@ -0,0 +1,219 @@
+// SPDX-FileCopyrightText: Copyright (c) 2025 Peter Magnusson <me@kmpm.se>
+// SPDX-License-Identifier: MIT
+package zeromq
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	gzmq4 "github.com/go-zeromq/zmq4"
+
+	"github.com/redpanda-data/benthos/v4/public/service"
+)
+
+func zmqInputNConfig() *service.ConfigSpec {
+	return service.NewConfigSpec().
+		Stable().
+		Categories("Network").
+		Summary("Consumes messages from a ZeroMQ socket.").
+		Description(`
+
+This is a native Go implementation of ZeroMQ using the go-zeromq/zmq4 library. ZMTP protocol is not supported.
+There is a specific instructions for C builds containing the original zmq4 component.`).

```

Comment:
Nitpicking here, but "There are ..."

We might need to include these instructions somewhere or reference them

Reply:
Sorry, thanks, fixed. English is not my native language.